l10n_nl_tax_statement_icp/README.rst

```diff
@@ -47,6 +47,18 @@ To configure this module, you need to:
 
 #. Follow the configuration steps as described in *l10n_nl_tax_statement* and set the tag *3b omzet* needed for this report.
 
+On the ICP Statement, the amounts will be split up into an amount for products
+and an amount for services. The Dutch chart of accounts in Odoo provides
+a separate set of taxes for services which ensure that the amounts are split
+up properly here as well. If you are migrating from a legacy situation or you
+do not want to apply separate taxes to service products, you can set a *System
```

Comment:
@yung-wang you are right, that's something also explained in the readme:

> If you are migrating from a legacy situation or you do not want to apply separate taxes to service products, you can set ...

What do you propose? Do you think this explanation in the readme is not enough?

Reply:
@astirpe The readme is good. Maybe in the future the migrated customer can adapt to the new flow and stick closer to how standards works. So for now, i approve this.